module/zfs/dsl_scan.c

```diff
@@ -4433,6 +4428,9 @@ ZFS_MODULE_PARAM(zfs, zfs_, max_async_dedup_frees, ULONG, ZMOD_RW,
 ZFS_MODULE_PARAM(zfs, zfs_, free_bpobj_enabled, INT, ZMOD_RW,
 	"Enable processing of the free_bpobj");
 
+ZFS_MODULE_PARAM(zfs, zfs_, scan_blkstats, INT, ZMOD_RW,
+	"Enable block statistics calculation during scrub");
```

Comment:
Add to `man4/zfs.4`, or alternately not make this user tunable at all.  I doubt anyone but a ZFS developer would end up turning it on in since there's no way to grab the stats without a debugger or more code changes.

Reply:
I left it out of man page because it was not exposed anyhow.  I was thinking to do it, but there are too many numbers, and I haven't decided how to format them.  Some dtrace script could get it though, so I think having tunable should be better than not.

Reply:
That works for me.  If we do end up making the stats visible, maybe as a kstat, we can document it then.